### DISCUSSION TITLE
What are zones (e.g. zone 2, zone 3) ?

### DISCUSSION THREAD

**Discussion by ghost** [microsoft/winget-cli#661] [Q&A] [answered]
I saw in the change log that was mentioned:
"Trusted sources (like the Microsoft Community Package Manager Repository) will have downloaded files **set to Zone 2**. Other sources will have be **set to Zone 3**, and the file will be run through all local Anti-virus (AV) software by attachment execution services."

so I wanted to ask what are the zones exactly?

**Comment by DonEstefan** [comment, marked as answer]
> A little known feature of NTFS is something called Alternate Data Streams, or ADS [...] It allows for storage of metadata for files residing on the filesystem. [...] This can include what zone the file was downloaded from, in this instance referring to the **Internet Properties zones** (Internet, Local Intranet, Trusted, etc) [...] 
> Zone Id | Zone Name
> -- | --
> 0 | My Computer
> 1 | Local Intranet
> 2 | Trusted Sites
> 3 | Internet
> 4 | Restricted Sites

Taken from https://xkln.net/blog/identifying-the-source-of-downloaded-files/
